### PR TITLE
Add index pages to all DBaaS services

### PR DESCRIPTION
**Pull request by tobru**

### CODE REVIEW

**Comment by wejdross** [comment]
LGTM

File: docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Please find more about the service description including pricing in our https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html[product documentation site].
+
+TIP: Get started with xref:exoscale-dbaas/redis/create.adoc[Creating a Redis instance].
+
+== Notes
+
+Some important notes about this service:
+
+* Connections are only possible using authenticated TLS encrypted connections.
```
Comment (zugao) [inline docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc:13]: Redis does not expose a certificate upon creation.

Comment (tobru) [reply docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc:13]: Ah true, I should add a few words about the ca.crt secret and on Redis that it's Let's Encrypt. Will add that...

Comment (zugao) [reply docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc:13]: Also FYI we were able to connect to all services without the ca certificate

Comment (tobru) [reply docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc:13]: > Also FYI we were able to connect to all services without the ca certificate.

How do you mean that? IMHO that depends on the application if a verification of the self-signed certificate is made or not.

Comment (zugao) [reply docs/modules/ROOT/pages/exoscale-dbaas/redis/index.adoc:13]: Never mind my last comment, I think I got smth wrong.

**Pull request merged**